recipes/zulu-openjdk/all/conanfile.py

```diff
@@ -1,19 +1,17 @@
-import os
-
 from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
 from conan.tools.files import get, copy
-
+import os
 
 class ZuluOpenJDK(ConanFile):
```

Comment:
Missing min conan version check?

Reply:
@RubenRBS 
Thank you for your comment.
I add `required_conan_version`.